src/foam/nanos/u2/navigation/SignUp.js

```diff
@@ -95,6 +95,15 @@ foam.CLASS({
         class: 'foam.u2.TextField',
         placeholder: 'Staff accountant'
       },
+      validationPredicates: [
+        {
+          args: ['department'],
+          predicateFactory: function(e) {
+            return e.NEQ(foam.nanos.u2.navigation.SignUp.DEPARTMENT, '');
+          },
+          errorString: 'Please select a Job Title'
```

Comment:
Is this a selection/drop-down?  as that is what 'select' suggests, while 'enter' suggest free form entry.

Reply:
this isn't  a selection/drop-down. just an text input that only check if this section is empty or not

Reply:
^ I agree about 'enter' instead of 'select', but want to add that "job title" shouldn't be capitalized

Reply:
Ok, chang made